includes/Settings.php

```diff
@@ -306,15 +306,17 @@ private static function initLegacyMapping( &$configuration ) {
 		// output notices
 		$GLOBALS['smwgDeprecationNotices'] = array(
 			'notice' => array(
-				'smwgAdminRefreshStore' => 'deprecated in 2.5, to be removed in 3.1'
+				'smwgAdminRefreshStore' => '3.1.0',
+				'smwgQueryDependencyPropertyExemptionlist' => '3.1.0',
+				'smwgQueryDependencyAffiliatePropertyDetectionlist' => '3.1.0'
```

Comment:
I added these to to provide a perspective on when they will be removed. To remove at the same time as the first setting listed seemed ok to me.